themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -134,9 +136,11 @@
         {block name='form_field_item_password'}
           <div class="input-group js-parent-focus">
             <input
+              id="field-{$field.name}"
               class="form-control js-child-focus js-visible-password"
               name="{$field.name}"
               title="{l s='At least 5 characters long' d='Shop.Forms.Help'}"
+              aria-label="{l s='Password input of at least 5 characters long' d='Shop.Forms.Help'}"
```

Comment:
```suggestion
              aria-label="{l s='Password input of at least 5 characters' d='Shop.Forms.Help'}"
```

Reply:
I think you forgot to remove the "long" at the end. :) Apart from this, wording lgtm.